packages/next/src/build/index.ts

```diff
@@ -591,9 +591,15 @@ async function writeFunctionsConfigManifest(
   distDir: string,
   manifest: FunctionsConfigManifest
```

Comment:
ouch

i think https://github.com/vercel/next.js/pull/86744 fixes the non-determinism (or at least one source of it) what do you think about that solution?